src/main/resources/anniv5Resources/localization/eng/runicpack/Powerstrings.json

```
@@ -0,0 +1,15 @@
+{
+  "${ModID}:ObeliskPower": {
+    "NAME": "Power of the Obelisk",
+    "DESCRIPTIONS": [
+      "When you change #yStances, Channel a random #yOrb.",
+      "When you change #yStances, Channel #b",
+      " random #yOrbs."
+    ]
+  },"${ModID}:AttackFocusLoss": {
+  "NAME": "Runic Concentration",
+  "DESCRIPTIONS": [
+    "When you play an Attack, lose #b1 #yFocus and reduce by 1."
```

I believe Attack should be highlighted here to be consistent with base game powers like Rage.
```suggestion
    "When you play an #yAttack, lose #b1 #yFocus and reduce by 1."
```